GVRf/Framework/framework/src/main/java/org/gearvrf/IRenderBundle.java

```diff
@@ -23,4 +23,9 @@
     GVRRenderTexture getPostEffectRenderTextureA();
 
     GVRRenderTexture getPostEffectRenderTextureB();
+    GVRRenderTarget getEyeCaptureRenderTarget();
+    GVRRenderTarget getDaydreamRenderTarget();
```

Comment:
Backend-specifics do not belong to the framework. Please move this to the daydream backend.